tests/integration/targets/iso_info/tasks/test.yml

```diff
@@ -26,10 +26,10 @@
   ansible.builtin.assert:
     that:
       - result.hcloud_iso_info | list | count == 1
-      - result.hcloud_iso_info[0].id == "{{ hcloud_iso_id }}"
-      - result.hcloud_iso_info[0].name == "{{ hcloud_iso_name }}"
-      - result.hcloud_iso_info[0].architecture == "{{ hcloud_iso_architecture }}"
-      - result.hcloud_iso_info[0].type == "{{ hcloud_iso_type }}"
+      - result.hcloud_iso_info[0].id == hcloud_iso_id
```

Comment:
This line fails in CI, not sure why.

Reply:
I am on a fix. It's a type problem string != int